src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  flex-shrink: 0;
+  width: get-calendar-width();
+
+  &__day {
+    @include flex-center;
+    @include square($square-size);
+    background-color: #eee;
+    border: $border-width solid #000;
+    transition: transform 0.5s, background-color 0.5s;
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(2n + #{$i}) {
```

2n is redundant here, you can easly write:
```suggestion
      &:nth-child(#{$i}) {
```

---

Really weird. I don't know why i did that. It mus have been late hahaha

---

That is what Code Reviews are for